dotnet/src/SemanticKernel/Services/AIServiceProvider.cs

```diff
@@ -4,7 +4,7 @@
 using System.Collections.Generic;
 
 namespace Microsoft.SemanticKernel.Services;
-internal class AIServiceProvider : NamedServiceProvider<IAIService>, IAIServiceProvider
+public class AIServiceProvider : NamedServiceProvider<IAIService>, IAIServiceProvider
```

Comment:
similar comment about public API, long term maintainability

Reply:
The change can be appreciated by .net developers who may want/need to use SK SDK in an advanced way by creating Kernel manually without KernelBuilder, why not? Anyway, I am open to discussing any concerns regarding this approach.